### PR TITLE
Do not check parameter bounds if parameter is not estimated

### PR DESCRIPTION
**Pull request by elbaraim**
If a parameter is fixed to a X value, petab checks for parameter bounds. This is not necessary and covered in this commit.

### CODE REVIEW

File: petab/lint.py [hunk 1]
```diff
@@ -172,11 +172,12 @@ def check_parameterBounds(parameter_df):
     in the parameter table.
     """
     for element in range(len(parameter_df['lowerBound'])):
-        if not parameter_df['lowerBound'][element] \
-                <= parameter_df['upperBound'][element]:
-            raise AssertionError(
-                f"lowerbound larger than upperBound in parameterId "
-                f"{parameter_df.index[element]}.")
+        if int(parameter_df['estimate'][element]) is True:
```
Comment (dweindl) [inline petab/lint.py]: `int(*) is True` is never true. I think `1` is currently used.

Check `'estimate' in parameter_df` first

File: petab/lint.py [hunk 2]
```diff
@@ -172,11 +172,12 @@ def check_parameterBounds(parameter_df):
...
+            if not parameter_df['lowerBound'][element] \
+                    <= parameter_df['upperBound'][element]:
+                raise AssertionError(
+                    f"lowerbound larger than upperBound in parameterId "
```
Comment (dweindl) [inline petab/lint.py]:
```suggestion
                    f"lowerBound greater than upperBound for parameterId "
```

File: petab/lint.py (hunk 1 above)
Comment (elbaraim) [reply petab/lint.py]: in `check_parameter_df` it is already called first `assert_parameter_estimate_is_boolean(df)` and then `check_parameter_bounds(df)`

Comment (elbaraim) [reply petab/lint.py]: If you do int('1') is resulting in True
`>>> int('1') == True
True`

File: petab/lint.py (hunk 2 above)
Comment (elbaraim) [reply petab/lint.py]: updated :+1:

File: petab/lint.py (hunk 1 above)
Comment (dweindl) [reply petab/lint.py]: Yes, but `x == True` does not imply that `x is True == True`. The latter also checks for matching types.

Comment (elbaraim) [reply petab/lint.py]: I replaced "is" with "=="

**Comment by yannikschaelte** [comment]
is the target really master? I guess it's for develop, right? that can be changed at the top of the page.

File: petab/lint.py [hunk 3]
```diff
@@ -171,11 +171,12 @@ def check_parameter_bounds(parameter_df):
     in the parameter table.
     """
     for element in range(len(parameter_df['lowerBound'])):
-        if not parameter_df['lowerBound'][element] \
-                <= parameter_df['upperBound'][element]:
-            raise AssertionError(
-                f"lowerBound larger than upperBound for parameterId "
-                f"{parameter_df.index[element]}.")
+        if int(parameter_df['estimate'][element]) == True:
```
Comment (dweindl) [inline petab/lint.py]:
```suggestion
        if int(parameter_df['estimate'][element]):
```

**Pull request merged**